### PR TITLE
feat(.vscode): add remix.code-snippets

### PR DESCRIPTION
**Pull request by hakimLyon**
## Screenshots

https://github.com/epicweb-dev/epic-stack/assets/64654633/6543eb0b-6505-4bb5-9704-f2931b33b43d

### CODE REVIEW

File: .vscode/remix.code-snippets [hunk 1]
```diff
@@ -0,0 +1,76 @@
+{
+	"loader": {
+		"prefix": "/ loader",
```
Comment (kentcdodds) [inline .vscode/remix.code-snippets:3]:
```suggestion
		"prefix": "/loader",
```

**Review by kentcdodds — COMMENTED** [review]
Thanks! Is there a way to limit these snippets to only apply in ts/js/tsx/jsx files?

File: .vscode/remix.code-snippets (hunk 1 above)
Comment (hakimLyon) [reply .vscode/remix.code-snippets:3]: Yes. In Visual Studio Code, for example, you can do this by adding a "scope" field with the desired language identifiers. 
```json
		"scope": "typescriptreact,javascriptreact,typescript,javascript",
```

**Comment by hakimLyon** [comment]
> Thanks! Is there a way to limit these snippets to only apply in ts/js/tsx/jsx files?

Yes. In Visual Studio Code, for example, you can do this by adding a "scope" field with the desired language identifiers. 
```json
		"scope": "typescriptreact,javascriptreact,typescript,javascript",
```

**Review by kentcdodds — APPROVED** [review]
Awesome! Thanks 🙏

**Pull request merged**